docs/client_reference.rst

```diff
@@ -2271,3 +2283,5 @@ Hierarchy of exceptions
   * :exc:`ClientPayloadError`
 
   * :exc:`InvalidURL`
+
+    * :exc:`InvalidRedirectURL`
```

Comment:
This looks overindented
```suggestion
  * :exc:`InvalidRedirectURL`
```

Reply:
It's a subclass, indentation is correct.

Reply:
Can we resolve this conversation?

Reply:
Yes, marking it as resolved.